fastlane/Fastfile

```diff
@@ -447,44 +451,30 @@ platform :ios do
 
     return if ENV['BUILDKITE_PULL_REQUEST'].nil?
 
-    post_installable_build_pr_comment(
-      app_name: 'simplenote-ios',
-      build_number: build_number,
-      url_slug: 'Simplenote-Installable-Builds'
-    )
+    post_installable_build_pr_comment
   end
 
   # Posts a comment on the current PR to inform where to download a given Installable Build that was just published to App Center.
   #
   # Use this only after `upload_to_app_center` as been called, as it announces how said App Center build can be installed.
   #
-  # @param [String] app_name The display name to use in the comment text to identify which app this Installable Build is about
-  # @param [String] build_number The App Center's build number for this build
-  # @param [String] url_slug The last component of the `install.appcenter.ms` URL used to install the app. Typically a sluggified version of the app name
-  #                          in App Center (e.g. `WPiOS-One-Offs`)
-  #
   # @called_by CI — especially, relies on `BUILDKITE_PULL_REQUEST` being defined
   #
-  def post_installable_build_pr_comment(app_name:, build_number:, url_slug:)
-    UI.message("Successfully built and uploaded installable build `#{build_number}` to App Center.")
+  def post_installable_build_pr_comment
```

Comment:
These installable builds changes are obviously unrelated to the TestFlight CI flow, but I noticed that the QR code was broken so I chucked them in here.